ginga/rv/plugins/Cursor.py

```diff
@@ -45,73 +45,47 @@ def __init__(self, fv):
 
         prefs = self.fv.get_preferences()
         self.settings = prefs.create_category('plugin_Cursor')
-        self.settings.add_defaults(share_readout=True)
+        self.settings.add_defaults(font_name=None, font_size=None)
         self.settings.load(onError='silent')
 
-        fv.add_callback('add-channel', self.add_channel_cb)
-        fv.add_callback('delete-channel', self.delete_channel_cb)
         fv.add_callback('field-info', self.field_info_cb)
         fv.set_callback('channel-change', self.focus_cb)
 
-        # TODO: let this become OUR setting
-        self.share_readout = self.fv.settings.get('share_readout', True)
         self.readout = None
+        self.gui_up = False
 
-    def _build_readout(self):
+    def build_gui(self, container):
         readout = Readout.Readout(-1, 24)
 
-        font_name = font_asst.resolve_alias('fixed', 'Courier')
-
         # NOTE: Special hack for certain platforms, otherwise the font
         # on the readout is too small
         macos_ver = platform.mac_ver()[0]
-        if len(macos_ver) > 0:
-            # Mac OS X
-            readout.set_font('monaco', 16)
-        elif toolkit.get_family().startswith('gtk'):
-            # Gtk
-            readout.set_font(font_name, 14)
-        else:
-            readout.set_font(font_name, 11)
-        return readout
-
-    def build_gui(self, container):
-
-        if self.share_readout:
-            self.readout = self._build_readout()
-            rw = self.readout.get_widget()
-            container.add_widget(rw, stretch=0)
-
-            channel = self.fv.get_channel_info()
-            if channel is not None:
-                self.focus_cb(self.fv, channel)
 
-    def add_channel_cb(self, viewer, channel):
-        fi = channel.fitsimage
+        font_size = self.settings.get('font_size', None)
+        if font_size is None:
+            if len(macos_ver) > 0:
+                # Mac OS X
+                font_size = 16
```

Comment:
I'll add a comment.

Reply:
@pllim, there is already a note a couple lines above:
```python
# NOTE: Special hack for certain platforms, otherwise the font
# on the readout is too small
```